### ISSUE TITLE
Dropout or not

### ISSUE THREAD

**Issue by DISAPPEARED13** [richzhang/PerceptualSimilarity#82]
if i use this loss function to calculate my model loss, should I set 'use_dropout' as False to get stable output? Thanks a lot. : )

**Comment by DISAPPEARED13** [comment]
> `use_dropout` should be `True`, since it was trained with dropout. Setting it in eval mode will make things stable.

Thanks for replying!! :D I use this for my model and it doesn't decrease, now I guess maybe I didn't set loss_model.eval() and calculate it properly. 
And.. I am sorry that I've got another problem to bother you, if I use 3D patch data to calculate it for example [2, 1, 32, 256, 256] should i split it into a shape like [1, 256, 256] of 2*32 arraies, get the loss each little array and sum it up? but I found this way will cost lots of GPU memory : (
